types/plugin.go

```diff
@@ -7,6 +7,15 @@ import (
 	"fmt"
 )
 
+// PluginInstallOptions holds parameters to install a plugin.
+type PluginInstallOptions struct {
+	Enable                bool
```

Comment:
This was "NoEnable" because it allowed one to control whether or not to enable the plugin immediately. If we want the default to be "enabled", this field should be `Disabled`.